phys/module_sf_clm.F

```diff
@@ -5120,9 +5120,10 @@ subroutine clmdrv(zgcmxy     ,forc_qxy   ,ps   ,forc_txy    ,tsxy  &
 ! URBAN CANOPY MODEL START - urban
 !--------------------------------------
 ! Input variables lsm --> urban
-
-          IF( IVGTYP(I,J) == 1 .or. IVGTYP(I,J) == low_density_residential .or. &
-              IVGTYP(I,J) == high_density_residential .or. IVGTYP(I,J) == high_intensity_industrial ) THEN
+ IF( IVGTYP(I,J) == 1    .or. IVGTYP(I,J) == LCZ_1 .or. IVGTYP(I,J) == LCZ_2 .or. &
```

Comment:
In other places, you use `IF( IVGTYP(I,J) == ISURBAN `. Is that more general than assuming urban ==1?